source/jsonnet/england-wales/household/blocks/visitor/visitor_usual_address_country.jsonnet

```diff
@@ -21,7 +21,7 @@ local rules = import 'rules.libsonnet';
         label: 'Current name of country',
         description: 'Enter your own answer or select from suggestions',
         mandatory: false,
-        suggestions_url: 'https://cdn.eq.census-gcp.onsdigital.uk/data/v1.0.0/countries-of-birth.json',
+        suggestions_url: '{suggestions_api_url}/countries-of-birth.json',
```

Comment:
Again, this naming is strange - This is the root of a url being passed, therefore `suggestions_url_root` might make more sense

Reply:
Done, thanks👍